client/extensions/woocommerce/components/basic-widget/index.js

```diff
@@ -3,22 +3,24 @@
  */
 import classNames from 'classnames';
 import React, { PropTypes } from 'react';
+import { noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 
-const BasicWidget = ( { buttonLabel, buttonLink, children, className, title } ) => {
+const BasicWidget = ( { buttonLabel, buttonLink, buttonClick, children, className, title } ) => {
```

Comment:
Would you mind renaming buttonClick to onButtonClick to make it clearer it is a function